README.md

```diff
@@ -0,0 +1,13 @@
+# &lt;palindrom-error-catcher&gt; [![Build Status](https://travis-ci.org/Palindrom/palindrom-error-catcher.svg?branch=gh-pages)](https://travis-ci.org/Palindrom/palindrom-error-catcher)
+
+---
+
+> Handles palindrom-client disconnection events and creates the needed UI to give the user control over them
+
+Custom Element that binds with [palindrom-client](https://github.com/Palindrom/palindrom-client) connection events and shows a simple UX that allows the user to interact with them. It is can be used as an example of designing your own error catcher.
```

Comment:
"shows UI", I think experience is not the thing to show, is a thing to feel.

Reply:
Right.